docs/number-input/number-input-spec.md

```diff
@@ -22,48 +18,66 @@ typora-copy-images-to: ../NumberInput
 
 ## Definition
 
-The **NumberInput** component improves upon the native `<input type="number">` by providing ability to customize the stepper arrows design, a common React+Typescript API, and working out the kinks of native implementations.
-
+**Number Input** allows  the user to enter a number within a certain range.
 
+Number Input improves upon the native `<input type="number">` by providing the ability to customise stepper arrows design, a common React+Typescript API, and working out the kinks of native implementations.
 
 ## Elements
 
-![elements](./elements.png)
+![elements](./assets/elements.png)
 
-The NumberInput consists of a native input and a Stepper subcomponent, which provides customisable Up and Down buttons. Also, you can add elements into it in the prefix and suffix slots, by passing the component children with the corresponding data-slot attribute.
+The **Number Input** functions as native `<input type="time">` component with additional ability to customise stepper and stepper arrows (using `::stepper`, `::stepper::up` ,`::stepper::down` subcomponents). Also number input can be customised by passing node elements into `prefix` and `suffix` properties.
```

Comment:
input="time" - seems like a mistake has crept in :)

Reply:
Holy cow! Yep, my bad. Fixed.